index.js

```diff
@@ -583,7 +621,12 @@ function getMetadata (opts) {
       container: undefined,
       kubernetes: undefined
     },
-    labels: opts.globalLabels
+    labels: opts.globalLabels,
+    cloud: opts.cloud
```

Comment:
This would allow to pass the cloud metadata directly if there is ever a need for that.